clay.h

```diff
@@ -76,38 +76,30 @@
 
 #define CLAY_STRING(string) (Clay_String) { .length = CLAY__STRING_LENGTH(string), .chars = (string) }
 
+int CLAY__ELEMENT_DEFINITION_LATCH = 0;
```

Comment:
I think this should be static so that "clay.h" can be included in multiple compilation units.

Reply:
Excellent point, and you've just reminded me to add a multi compilation unit test.

Reply:
Fixed in https://github.com/nicbarker/clay/pull/30/commits/e3dacd93709688036624fa50a28ffa51824236e1